deployment/helm/values.yaml

```diff
@@ -254,6 +366,27 @@ opcua:
     targetPort: 8083
     # protocol is the service protocol of the instance service
     protocol: TCP
+  # discovery defines a set of values for a opcua discovery handler DaemonSet
+  discovery: 
+    # enabled defines whether discovery handler pods will be deployed in a slim Agent scenario
+    enabled: false
+    image:
+      # repository is the container reference
+      repository: ghcr.io/deislabs/akri/opcua-discovery
+      # tag is the container tag
+      # opcua.yaml will default to v(AppVersion)-dev
+      tag:
+      # pullPolicy is the pull policy
+      pullPolicy: ""
+    # useNetworkConnection specifies whether the discovery handler should make a networked connection
+    # with Agents, using its pod IP address when registering
+    useNetworkConnection: false
+    # port specifies (when useNetworkConnection is true) the port on which the discovery handler advertises its discovery service
+    port: 10000
```

Comment:
it seems like this setting should be an agent setting rather than a per-DH setting.  (i'm imagining someone needing to specify a different port and needing to do a lot of typing if they have 3 DHs)

Reply:
also, is 10000 a good port to choose as a default value? (i have no idea, just making sure we think about it)

Reply:
This is the port that is used if the DiscoveryHandler devices to `discovery` over ip instead of UDS. The Discovery Handler specifies its address when it registers with the Agent, so it needs to know the port (that has been exposed on its Pod) rather than the Agent. As for 10000, not sure. @jiria, do you know what a good default port is?

Reply:
my suggestion was more about values.yaml than the existence of port as a parameter.

it seems like there should be ONLY an agent.registrationPort setting that everything else uses ... NOT an agent.registrationPort AND a  debugEcho.discovery.port AND a foo.discovery.port AND a bar.discovery.port.

if there is only one place to make the change, it makes it less likely that someone will end up with agent.registrationPort=10001 and foo.discovery.port=10002.

Reply:
Sorry, `registrationPort` was a dinosaur from before i made registration purely UDS. It should not exist. Also, I believe that it shouldnt matter if discoveryHandlers have the same port, since the IP addresses is POD_IP not HOST_IP.

Reply:
ok :)

Reply:
removed `registrationPort` in 0fc1223b288